.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: node_js
+
+node_js:
+  - "stable"
```

Comment:
I think with "stable"  Travis will run on the latest stable Node.js version .
It's preferred to stick to one Node.js version on both Travis and your local machines. 
so replace `"stable"`  with `"version number"`
also specify the Node.js version in your` package.json`  like this : 
 ```
 "engines": {
    "node": "the version number"
  }
```

Reply:
specifying  
```
"engines": {
   "node": "the version number"
 }
```
is for heroku